apps/content/content/docs/contract-first.mdx

```diff
@@ -89,20 +90,24 @@ With your contract defined, implement the server logic:
 import { os, ORPCError } from '@orpc/server'
 import { contract } from 'examples/contract'
 
-export const pub /** public access */ = os.contract(contract) // Ensure every implement must be match contract
-export const authed /** require authed */ = os
-  .use((input, context, meta) => /** put auth logic here */ meta.next({}))
+export const publicProcedure = os.contract(contract) // Ensure every implement must be match contract
```

Comment:
contain `procedure` can be mislead, since this can use for `middleware`, `router`, ...

Reply:
we can call it something like publicRoute and authRoute instead perhaps?

Reply:
I think we should keep the original. In the future, it might serve broader purposes, so restricting its meaning within this name could be misleading.